### PR TITLE
sqltable compute nan frequency

### PR DESCRIPTION
**Pull request by noahnovsak**
##### Issue
`__get_nan_frequency' uses X and Y arrays, so SqlTables need an alternative implementation.

### CODE REVIEW

**Comment by markotoplak** [comment]
@lanzagar or @PrimozGodec: can one of you please check this PR? Thanks!

File: Orange/tests/sql/test_sql_table.py [hunk 1]
```diff
@@ -764,6 +764,20 @@ def test_list_tables_with_schema(self):
             with self.backend.execute_sql_query("DROP SCHEMA IF EXISTS orange_tests CASCADE"):
                 pass
 
+    @dbt.run_on(["postgres"])
```
Comment (PrimozGodec) [inline Orange/tests/sql/test_sql_table.py:767]: It should be also tested on mssql

```suggestion
    @dbt.run_on(["postgres", "mssql"])
```

File: Orange/data/sql/table.py [hunk 2]
```diff
@@ -654,3 +654,19 @@ def _execute_sql_query(self, query, param=None):
 
     def checksum(self, include_metas=True):
         return np.nan
+
+    def __get_nan_frequency(self, columns):
+        nans = 0
+        for column in columns:
+            col_field = column.to_sql()
+            query = self._sql_query(["COUNT(*)"],
+                                    filters=[f"{col_field} IS NULL OR {col_field} = 'NaN'"])
```
Comment (PrimozGodec) [inline Orange/data/sql/table.py:663]: Checking NULL is sufficient. Checking `"nan"` even fail on MSSQL when column is float.
```suggestion
        query = self._sql_query(["COUNT(*)"], filters=[f"{col_field} IS NULL"])
```

Comment (PrimozGodec) [reply Orange/data/sql/table.py:663]: Anyway, can we make it in one query to speed the process up? For example with this: https://stackoverflow.com/a/16528738/3551700

**Review by PrimozGodec — CHANGES_REQUESTED** [review]

**Comment by PrimozGodec** [comment]
Additionally (not connected with your changes but required to insert NULL values in database correctly) `Orange.test.base` lines 280-285 should be changed to:
```
        insert_values = ", ".join(
            "({})".format(
                ", ".join(
                    "NULL" if pd.isna(v) else "'{}'".format(v)
                    for v, t in zip(row, sql_column_types)
                )
            )
            for row in data
        )
```
I replaced v is None with pd.isna(v) to correctly handle None and nan values.

**Comment by PrimozGodec** [comment]
@noahnovsak thank you for patience.

**Pull request merged**